src/gui/qgsexpressionaddfunctionfiledialog.cpp

```diff
@@ -0,0 +1,67 @@
+/***************************************************************************
+    qgsexpressionaddfunctionfiledialog.cpp
+    ---------------------
+    begin                : May 2024
+    copyright            : (C) 2024 by Germán Carrillo
+    email                : german at opengis dot ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include <QPushButton>
+#include <QStandardItemModel>
+#include "qgsexpressionaddfunctionfiledialog.h"
```

Comment:
Are the two Qt includes necessary here? I'd have assumed the .ui would have taken care of this. In any case, let's re-order to have QGIS includes first if needed :)
```suggestion
#include "qgsexpressionaddfunctionfiledialog.h"

#include <QPushButton>
#include <QStandardItemModel>
```

Reply:
Both are needed here, I've fixed the order according to your suggestion, thanks.